tools/stub_generator/npp.json

```
@@ -14,6 +14,16 @@
          "return_type":"NppLibraryVersion *",
          "not_found_error":"nullptr" },
       "nppiColorTwist32f_8u_C3R": {},
-      "nppiColorTwist32f_8u_C1R": {}
+      "nppiColorTwist32f_8u_C1R": {},
+            "nppiRemap_64f_C1R_Ctx": {},
+      "nppiRemap_64f_C3R_Ctx": {},
```

```suggestion
      "nppiColorTwist32f_8u_C1R": {},
      "nppiRemap_64f_C1R_Ctx": {},
      "nppiRemap_64f_C3R_Ctx": {},
```
The indentation looks off

---

Fixed